scriptmodules/supplementary/esthemes.sh

```diff
@@ -185,6 +185,7 @@ function gui_esthemes() {
         'waweedman Venom'
         'waweedman Spider-Man'
         'blowfinger77 locomotion'
+        'CrashCortez Crashstheme'
```

Comment:
For this to work, you'll have to name your theme Github repository `es-theme-crashstheme`.

Reply:
ill fix that now, thank you.